src/parser.rs

```diff
@@ -378,6 +378,7 @@ impl<'parser> Parser<'parser> {
                 } else {
                     // All block comment lines are indented at least beyond the "/*", so strip the
                     // indent and re-indent when formatting.
+                    let line_count = content.lines().count();
```

Comment:
This line shouldn't be here. There is a warning about this, but you may have overlooked it due to all of the errors about missing docs.

Do you want to take a stab at documenting the new public APIs now?

Thanks,
Rich

Reply:
Same issue as before, thanks for noticing. Yeah, I'll get to work at that documentation right now!